src/overlays/actors/ovl_En_Ginko_Man/z_en_ginko_man.c

```diff
@@ -512,7 +512,7 @@ void EnGinkoMan_SetupBankAward(EnGinkoMan* this) {
 
 void EnGinkoMan_BankAward(EnGinkoMan* this, GlobalContext* globalCtx) {
     if (Actor_HasParent(&this->actor, globalCtx)) {
-        // ? when would bank have a parent?
+        // Parent is the player when starting to receive the award
```

Comment:
might just be me, but I feel like the comment is not needed at all. the parent/child relationship for get items is a common thing and doesnt really need to be singled out here

Reply:
As someone who has decomped and attempted to document multiple actors doing this I didn't know what was happening (thats my comment being fixed) until Zoey pointed it out. I don't feel this was intuitive at all and I've actually picked and poked at this actor trying to document it and still missed it. 

I feel most people visiting this actor wouldn't know what this code did if they didn't already have a deep understanding of how the game operates (which documentation should help reduce the requirement for), and while not unique to this actor, I feel the comment adds context to what the code is doing. 

It should probably be reworded to be less generic "this is what this kind of code does" and more actor specific, something like 
`// Player is parent: receiving wallet upgrade or 5 Rupees`